api/data_refinery_api/tests.py

```diff
@@ -281,6 +285,18 @@ def test_sample_pagination(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.json()["results"][0]["title"], "123")
 
+    def test_organism_pagination(self):
+        response = self.client.get(reverse("organisms", kwargs={"version": API_VERSION}))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(len(response.json()["results"]), 25)
+
+        # First organism on second page should be TEST_ORGANISM_25
+        response = self.client.get(
+            reverse("organisms", kwargs={"version": API_VERSION}), {"offset": 25}
+        )
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        self.assertEqual(response.json()["results"][0]["name"], "TEST_ORGANISM_25")
```

Comment:
Can you test that the second page has only 1 result?

Reply:
Do the other tests make more organisms or something? If you make 26 organisms and then make the offset 25, shouldn't there only be 1 on the second page?